queries.sql

```diff
@@ -50,4 +50,69 @@ FROM animals
 INNER JOIN owners 
  ON animals.owner_id = owners.id
 GROUP BY owners.full_name
-ORDER BY num DESC LIMIT 1;
\ No newline at end of file
+ORDER BY num DESC LIMIT 1;
+
+
+-- join table for visits queries
+
+SELECT animals.name FROM animals JOIN visits ON visits.animals_id = animals.id 
+JOIN vets ON vets.id = visits.vets_id WHERE vets.name = 'William Tatcher' 
+ORDER BY visits.date DESC LIMIT 1;
+
+
+SELECT COUNT(animals.name) FROM animals JOIN visits ON visits.animals_id = animals.id 
+JOIN vets ON vets.id = visits.vets_id WHERE vets.name = 'Stephanie Mendez'; 
+
+
+SELECT * FROM vets LEFT JOIN specializations ON specializations.vets_id = vets.id 
+LEFT JOIN species ON species.id = specializations.species_id; 
+
+
+SELECT * FROM animals JOIN visits ON visits.animals_id = animals.id 
+JOIN vets ON vets.id = visits.vets_id WHERE vets.name = 'Stephanie Mendez' AND
+visits.date BETWEEN '2020-5-1' AND '2020-8-30'; 
+
+
+
+SELECT COUNT(animals.name) AS num, animals.name
+FROM animals 
+INNER JOIN visits 
+ON visits.animals_id = animals.id
+INNER JOIN vets ON visits.vets_id = vets.id
+GROUP BY animals.name
+ORDER BY num  LIMIT 1;
```

Comment:
- [ ] You can try and make this result in a descending manner so you get the animal with the most visits. in this way, you get only one result but you don't know if it is the one with the most or fewer visits